code-of-conduct.html

```diff
@@ -0,0 +1,134 @@
+<html>
+<head>
+<title>satRday Code of Conduct</title>
+</head>
+<body>
```

Comment:
I also think it would be worthwhile to add a button linking back to the main page (again the below relies on the bootstrap CSS):
```
<body>
<div class="col-md-10 col-md-offset-1">
<a class="btn btn-primary" href="https://capetown2018.satrdays.org">Back to main page</a>
<h1> satRday Code of Conduct / Anti-Harassment Policy </h1>
...etc
```